common/datasets/librispeech.py

```diff
@@ -324,7 +324,7 @@ def get_special_lemma_lexicon():
     return lex
 
 
-def get_bliss_lexicon(subdir_prefix=""):
+def get_bliss_lexicon(subdir_prefix="", use_stress_marker=False):
```

Comment:
docstring for stress marker is missing, and this should be before the `subdir_prefix` (I think it is a good convention to have it last, but we actually need to discuss at some point)

Reply:
I would still move `use_stress_marker` to be the first argument, because this is the one that influence the output.